package.json

```diff
@@ -86,6 +86,7 @@
     "resize-observer-polyfill": "^1.5.0",
     "toposort": "^1.0.3",
     "url": "^0.11.0",
+    "uuid": "^8.2.0",
```

Comment:
This lib makes easy to generate unique identifiers. I could probably just use a variable that increment on every call to the `display` action, but this uuid lib is very well supported, relatively small and it will probably have other uses in the long run.